File: Arithmetic.java
import java.util.Scanner;

public class Arithmetic {

    public static void main(String[] args) {

        Scanner input = new Scanner(System.in);

        System.out.print("Enter an integer one ");

        float number1 = input.nextFloat();

        System.out.print("Enter an integer two ");

        float number2 = input.nextFloat();

        float square1 = number1 * 2;

        float square2 = number2 *2;

	float different = square1 - square2;

        System.out.println("square of integer one : "+ square1 );
	System.out.println("square of integer two :  "+square2);
	System.out.println("The different of square1 and square2 is:" + different);



    }


}

File: Arithmetic.py
number1 = int(input("Enter a firstnumber:"))
number2 = int(input("Enter second number:"))
number3 = int(input("Enter third number"))

sum = number1 + number2 + number3
print("the sum of", sum)

average = sum/3
print("the average is",average)

product = number1 * number2 * number3
print("the product is", product)

if number1 > number2 and number1 > number3:
    print(number1)
if number2 > number1 and number2 > number3:
    print(number2)
if number3 > number1 and number3 > number2:
    print(number3)

    if number1 < number2 and number1 < number3:
        print(number1)
        if number2 < number1 and number2 < number3:
            print(number2)

            if number3 < number1 and number3 < number1:
                print(number3)



